app/resources/games/Quake3/GameConfig.cfg

```diff
@@ -26,29 +26,34 @@
             "name": "Clip brushes",
             "attribs": [ "transparent" ],
             "match": "surfaceparm",
-            "pattern": "playerclip"
+            "pattern": "playerclip",
+            "ignore": ["common/caulk", ""]
```

Comment:
what's the `""` for?

Reply:
To ignore faces without any texture assigned to them; this sometimes happens in Q3 maps (e.g. moteof).